src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -8703,7 +8704,7 @@ private TypeWithState VisitConversion(
             bool canConvertNestedNullability = true;
             bool isSuppressed = false;
 
-            if (conversionOperand.IsSuppressed == true)
+            if (suppressed || conversionOperand.IsSuppressed)
```

Comment:
Could you share some details on the set up of the situation? I gather that we have two stacked conversions that are not part of the same conversion group and the operand is suppressed, but I don't understand why.
If this setup is expected, should we just dig through conversions here when looking for the `IsSuppressed` flag?